src/autofill-utils.js

```diff
@@ -275,7 +275,7 @@ const SUBMIT_BUTTON_REGEX = /submit|send|confirm|save|sign|log.?([io])n|buy|purc
 const isLikelyASubmitButton = (el) =>
     el.getAttribute('type') === 'submit' || // is explicitly set as "submit"
     /primary|submit/i.test(el.className) || // has high-signal submit classes
-    SUBMIT_BUTTON_REGEX.test(el.textContent || el.title) || // has high-signal text
+    SUBMIT_BUTTON_REGEX.test(el.textContent || el.title || el.value) || // has high-signal text
```

Comment:
`value` is an optional attribute, so on elements like links it will be `undefined`. What about adding a further fallback like
```suggestion
    SUBMIT_BUTTON_REGEX.test(el.textContent || el.title || el.value || '') || // has high-signal text
```